package.json

```diff
@@ -4,7 +4,8 @@
     "@discourse/lint-configs": "2.2.1",
     "ember-template-lint": "6.0.0",
     "eslint": "9.14.0",
-    "prettier": "2.8.8"
+    "prettier": "2.8.8",
+    "@babel/plugin-proposal-decorators": "^7.25.7"
```

Comment:
hmm, was this required to run linters?

Reply:
could you try with a fresh node_modules? (`rm -rf node_modules && pnpm i`)
and what lint command are you using?

Reply:
Ok, i think I have a fix in `@discourse/lint-configs` to make this more robust. I'm still curious why this is only affecting you though @awesomerobot. Could you share your pnpm version?

Reply:
Should have replied here 😅 I see the same error in vscode here